tensorflow/python/framework/device_spec.py

```diff
@@ -47,7 +47,7 @@ def _as_device_str_or_none(device_type):
 @tf_export("DeviceSpec", v1=[])
 class DeviceSpecV2(object):
   """Represents a (possibly partial) specification for a TensorFlow device.
-
+  
```

Comment:
This still shows up as a trailing whitespace

```
   """Represents a (possibly partial) specification for a TensorFlow device.
-
+  
   `DeviceSpec`s are used throughout TensorFlow to describe where state is stored
```

(from https://patch-diff.githubusercontent.com/raw/tensorflow/tensorflow/pull/34162.diff)